package.json

```diff
@@ -666,12 +666,12 @@
       "view/item/context": [
         {
           "command": "k8s.tekton.run.logs",
-          "when": "viewItem =~ /vsKubernetes\\.resource\\.(pipelinerun|taskrun)/i",
+          "when": "viewItem =~ /vsKubernetes\\.resource\\.(pipelinerun|pipelineRun|taskrun)/i",
```

Comment:
It looks like mistake `pipelinerun`  and `pipelineRun`, what the difference between them?

Reply:
To hide the cancel button I am using `pipelineRun`, I will switch the contextValue to `pipelineRun` when pipelinerun is finished.

Reply:
So, maybe `pipelineRunFinished` is more descriptive?

Reply:
we are using `pipelineRun` to one file in the editor. So I don't think `pipelineRunFinished` would be good to use here.

Reply:
OK, anyway having `pipelinerun` and `pipelineRun` is not good, you need to rename one of it.